### PR TITLE
changed web-api setup

### PR DESCRIPTION
**Pull request by KristianS93**
Updated the setup guide for the webapi.

### CODE REVIEW

File: docs/Getting_Started/setup.md [hunk 1]
```diff
@@ -156,27 +184,65 @@ Select the database tab, add an instance, and select MySQL. Use the connection s
...
+
+Another approach would be to change the launchsettings, specifically the ASPNETCORE_ENVIRONMENT to `LocalDocker`
+
+Now you should be able to develop in your IDE, run the API directly in the IDE, and utilize the Docker MySQL container. 
+
+### Running the web-API with Docker
+
+Start by downloading the Docker, this can be either the Docker Engine itself (this requires more knowledge of Docker), or the easier alternative Docker Desktop (Linux, MacOS and Windows) [Download Docker Desktop](https://www.docker.com/get-started/).
+
+Ensure that you cloned the web-API repoitory from this section [Clone the web-API](#clone-the-web-api-repository).
```
Comment (Schoogle) [inline docs/Getting_Started/setup.md]: Mindre stavefejl ved repoitory

File: docs/Getting_Started/setup.md [hunk 2]
```diff
@@ -156,27 +184,65 @@ Select the database tab, add an instance, and select MySQL. Use the connection s
...
-application again. It is possible to check the current version of api client your local weekplanner 
-uses in Android Studio by opening External Libraries → Dart Packages
+Next shutdown the web-api container within the application, in docker desktop this can be done, by clicking the stop icon. 
 
-NB: Do not push this change to pubspec unless you know what you are doing, and make sure to 
-change it back to develop before merging.
+Shown here:
+![stop container](./images/stopAPI.png)
+
+After doing that they api show have stopped and you should have the following:
```
Comment (Schoogle) [inline docs/Getting_Started/setup.md]: the api should have stopped?

File: docs/Getting_Started/setup.md (hunk 1 above)
Comment (KristianS93) [reply docs/Getting_Started/setup.md]: fixed

File: docs/Getting_Started/setup.md (hunk 2 above)
Comment (KristianS93) [reply docs/Getting_Started/setup.md]: fixed

**Review by andreaskrath — APPROVED** [review]
Incredible commitment to documenting the GIRAF journey!

I truly hope more people show your level of energy when working on open source software in the future!

**Review by JMyrtue — APPROVED** [review]

**Pull request merged**